<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userInterface</class>
 <widget class="QDialog" name="userInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KATodoList</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/AppIcon.png</normaloff>img/AppIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTableWidget" name="listOfTask">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>131</y>
     <width>720</width>
     <height>340</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>Task</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>End</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Check</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnSignOut_user">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(210, 212, 255);
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Sign out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditTask">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 212, 255);
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Edit Task</string>
   </property>
  </widget>
  <widget class="QLabel" name="toDoList">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Todo list&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddTask">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 212, 255);
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Add Task</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteTask">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 212, 255);
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Delete Task</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 212, 255);
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Save all</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
